lib/utils/__tests__/checkAgainstRule.test.mjs

```diff
@@ -87,27 +87,6 @@ describe('checkAgainstRule', () => {
 		expect(warnings[0].column).toBe(1);
 	});
 
-	/** @todo remove once all fixable rules have been been migrated from context.fix */
-	it('outputs fixed code when provided a context object', async () => {
```

Comment:
[ask] Don't we ensure the compatibility of `checkAgainstRule()`? 🤔

Reply:
The only way would be to test a plugin rule.

Reply:
Just opened PR #7833, which blocks this PR.